### PR TITLE
Update container subject type for test result

### PR DESCRIPTION
**Pull request by hluk**

### CODE REVIEW

**Comment by lslebodn** [comment]
Looks like (unit )tests will need to be updated based on gitlab checks
```
Chrome Headless 105.0.5195.125 (Linux x86_64) GreenwaveService testing getArtifactDecision calls getDecision correctly with a ContainerKojiBuild FAILED
	Expected spy getDecision to have been called with:
	  [ 'some-nvr', 'cvp_default', 'cvp', 'koji_build', true ]
	but actual calls were:
	  [ 'some-nvr', 'cvp_default', 'cvp', 'redhat-container-image', true ].
```

**Comment by lslebodn** [comment]
BTW the change is required because CVP finally manged to migrate from UMB messages for rpms to
messages for container images
* https://pagure.io/fedora-ci/messages/blob/master/f/schemas/redhat-container-image.test.complete.yaml

File: src/app/services/greenwave.service.spec.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ describe('GreenwaveService testing', () => {
       'waivers': []
     };
 
-    greenwaveService.getDecision('some-continaer', 'cvp_default', 'cvp', 'koji_build').subscribe(data => {
+    greenwaveService.getDecision('some-continaer', 'cvp_default', 'cvp', 'redhat-container-image').subscribe(data => {
```
Comment (ronnyhlim) [inline src/app/services/greenwave.service.spec.ts:46]: i know this was here prior to the MR, but wondering if it's worth changing `some-continaer` to `some-container`

**Comment by qixiang** [comment]
:+1: I'm not familiar with angular or ts, but this change looks good and the reason is clear, so I'm giving my +1 on this.

**Review by qixiang — APPROVED** [review]

**Comment by hluk** [comment]
Well, I can merge this but is anyone still able to deploy the change?

**Comment by qixiang** [comment]
> Well, I can merge this but is anyone still able to deploy the change?

I'll try to deploy this after it's merged, but I'll need your help to verify the change is deployed as expected.

**Comment by lslebodn** [comment]
> I'll try to deploy this after it's merged, but I'll need your help to verify the change is deployed as expected.

One can check internal deployment for the nvr postgresql-13-container-1-79.1664813213.

ATM it says: 2 of 5 required test results missing
But actually all test-results are provided and it should be green

https://p.e.r.c/kojibuild/postgresql-13-container-1-79.1664813213

**Comment by qixiang** [comment]
> One can check internal deployment for the nvr postgresql-13-container-1-79.1664813213.
> 
> ATM it says: 2 of 5 required test results missing But actually all test-results are provided and it should be green
> 
> https://p.e.r.c/kojibuild/postgresql-13-container-1-79.1664813213

Thank you for providing this verify approach, Estuary prod has been updated and it now shows "All required tests passed".

I rebuilt the `estuary-frontend` image with `oc` commands to deploy this change:
```
$ oc get bc
NAME               TYPE     FROM         LATEST
estuary-frontend   Source   Git@master   3
$ oc start-build bc/estuary-frontend
```

**Comment by lslebodn** [comment]
ty very much

**Pull request merged**